distributions/openhab/src/main/resources/bin/restore.bat

```diff
@@ -0,0 +1,24 @@
+@ECHO off
+SETLOCAL
+
+IF NOT [%~4]==[] IF NOT "%~4"=="true" IF NOT "%~4"=="false" GOTO printArgs
+
+SET autoConfirm=False
+IF "%~4"=="true" SET autoConfirm=True
+powershell -command "& { . .\restore.ps1; Restore-openHAB -OHDirectory %~1 -OHBackups %~2 -FileName %~3 -AutoConfirm $%autoConfirm% }"
```

Comment:
and again adding....
. .**_\runtime\bin_**\restore.ps1
...so that RC - RAA works.

Also need to check for %1 again.
`if [%1]==[] (powershell -command "& { . .\runtime\bin\restore.ps1; Restore-openHAB }") else (powershell -command "& { . .\runtime\bin\restore.ps1; Restore-openHAB -OHDirectory %~1 -OHBackups %~2 -FileName %~3 -AutoConfirm $%autoConfirm% }")`